### PR TITLE
FR lessons requests page creation -  closes #1431 

### LINKED ISSUE #1431: Lesson requests FR page
During #1358 meeting of August 30 we worked on the lesson requests EN and ES pages. This is a good occasion to start planning the creation of the FR page, so I open this ticket to start the discussion about setting a timeline for its publication and deciding on the type of original lessons the @programminghistorian/french-team wishes to call for.  

EN and ES pages can be found here: 
https://programminghistorian.org/en/lesson-requests 
https://programminghistorian.org/es/solicitud-lecciones

### PR DESCRIPTION
**Pull request by spapastamkou**
I am raising the pull request so that the @programminghistorian/french-team is able to work on the final version of the FR lessons requests page. 
Please @fdlaramee and @grandjeanmartin take a look when you are able to check the translated parts and the two additional requests in the end of the list. Fell free to modify directly or request modifications, additions etc. 
PS: no urgeny.

### CODE REVIEW

File: fr/appel-contributions.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# Appel à rédiger un tutoriel 
+
+Le comité éditorial de la version francophone du _Programming Historian_ est particulièrement intéressé à recevoir des propositions de tutoriels sur les sujets suivants. Merci de consulter aussi, pour plus d'informations, nos [consignes aux auteur(e)s](/fr/consignes-auteurs):
+
+1. How do you conduct spatial clustering of geographic data?
+
+    We’ve got a great set of introductory mapping lessons, and while they are great for teaching how to make nice visualisations, we’ve not yet branched deeply enough into more advanced analysis skills. One of the most useful is the application of clustering algorithms, which identify logical groups of individual points in geographic space. Useful for forming conclusions on anything from trade to migration. But like with all analyses, it’s a space (no pun intended) fraught with pitfalls for the uninitiated. We’re looking for a great introduction that highlights both the strengths and the challenges of this form of analysis.
```
Comment (mdlincoln) [inline fr/appel-contributions.md]: Is this still waiting to be translated? Or is English text here accidentally?

Comment (spapastamkou) [reply fr/appel-contributions.md]: Had left this for the end bcz had trouble with translating some terms. Done now:)

**Review by fdlaramee — CHANGES_REQUESTED** [review]
@spapastamkou Only a handful of suggestions:

Lesson 1: probably worth mentioning that the basic cartography lessons are in English only for the moment?

Lesson 3: "textes numériques structurées" contains a masculine noun and a feminine adjective

Lessons 4, 5 and 6: I still see the English titles and text along with the French versions.

Otherwise, all good as far as I'm concerned. Thanks!

**Comment by spapastamkou** [comment]
I am making minor changes in the text, it is not to be merged yet.

**Comment by spapastamkou** [comment]
@fdlaramee I think I made all requested changes. Please check if the final version is ok, and if it is the case feel free to merge.

I also replaced a dead link in [Transforming Data for Reuse and Re-publication with XML and XSL](https://programminghistorian.org/en/lessons/transforming-xml-with-xsl) and its ES translation because it hindered the travis build. I used the Internet Archive's link for the same page.

**Review by fdlaramee — APPROVED** [review]

**Pull request merged**